dts/riscv/bflb/bl60x.dtsi

```diff
@@ -151,5 +151,12 @@
 			compatible = "mmio-sram";
 			reg = <0x42030000 DT_SIZE_K(112)>;
 		};
+
+		efuse: efuse@40007000 {
```

Comment:
We need to use ascending order of address. This means that it is necessary to move the node between `pinctrl` and `uart0`.

```diff
		pinctrl: pin-controller@40000000 {
			...
		};

+		efuse: efuse@40007000 {
+			compatible = "bflb,efuse";
+			reg = <0x40007000 0x1000>;
+			status = "okay";
+			size = <128>;
+		};
+
		uart0: uart@4000a000 {
...
-		efuse: efuse@40007000 {
-			compatible = "bflb,efuse";
-			reg = <0x40007000 0x1000>;
-			status = "okay";
-			size = <128>;
-		};
	};
};
```

Reply:
Done